### DISCUSSION TITLE
Using app.head.scripts async loads the script twice?

### DISCUSSION THREAD

**Discussion by AndersNielsen85** [nuxt/nuxt#17262] [Questions]
I am trying to add a javascript trough app.head.scripts.

It seems like this script is loaded twice when using async (guessing 1 for server 1 for client?).
`export default defineNuxtConfig({
  app: {
    head: {
      script: [
        {
          id: 'dummy',
          src: '/dummy.js',
          async: true // Causes double load?
        }
      ]
    }
  }
})`

Removing the `async: true` seems to make it work.

In my crude example this results in an alert 2times instead of the desirable one.

Is there any way of avoiding this behaviour, while still retaining an async script load?

**Reproduction**
[https://stackblitz.com/edit/nuxt-starter-nna48v?file=nuxt.config.ts](https://stackblitz.com/edit/nuxt-starter-nna48v?file=nuxt.config.ts)

**Comment by mirumirumi** [comment]
Hi!
I have seen similar threads in the past. 
https://github.com/nuxt/framework/discussions/5107

However, the writing style could be changed a little more, so how about the following example?

```html
// app.vue

<template>
  <div id="app">Your great app</div>
</template>

<script setup lang="ts">
if (process.server) {
  useHead({
    script: [
      { src: "/dummy.js", async: true },
    ],
  })
}
</script>
```
